src/KeyboardLayouts/ImprovedKeylayoutsIT.h

```diff
@@ -0,0 +1,304 @@
+	KEY_4|MOD_LEFT_SHIFT,    		// $
+	KEY_5|MOD_LEFT_SHIFT,    		// %
+	KEY_6|MOD_LEFT_SHIFT,    		// &
+	KEY_IT_QUOTE,          	// '
+	KEY_8|MOD_LEFT_SHIFT,    		// (
+	KEY_9|MOD_LEFT_SHIFT,    		// )
+	KEY_IT_PLUS|MOD_LEFT_SHIFT,    	// *
+	KEY_IT_PLUS,				    // +
+	KEY_COMMA,          	// ,
+	KEY_IT_MINUS,          	// -
+	KEY_PERIOD,          	// .
+	KEY_7|MOD_LEFT_SHIFT, 	// /
+	KEY_0,          		// 0
+	KEY_1,          		// 1
+	KEY_2,          		// 2
+	KEY_3,          		// 3
+	KEY_4,          		// 4
+	KEY_5,          		// 5
+	KEY_6,          		// 6
+	KEY_7,          		// 7
+	KEY_8,          		// 8
+	KEY_9,          		// 9
+	KEY_PERIOD|MOD_LEFT_SHIFT,		// :
+	KEY_COMMA|MOD_LEFT_SHIFT,       // ;
+	KEY_IT_LT_GT,				    // <
+	KEY_0|MOD_LEFT_SHIFT,          	// =
+	KEY_IT_LT_GT|MOD_LEFT_SHIFT,    // >
+	KEY_IT_QUOTE|MOD_LEFT_SHIFT,   	// ?
+	KEY_IT_OGRAVE|MOD_RIGHT_ALT,   	// @
+	KEY_A|MOD_LEFT_SHIFT,      		// A
+	KEY_B|MOD_LEFT_SHIFT,      		// B
+	KEY_C|MOD_LEFT_SHIFT,      		// C
+	KEY_D|MOD_LEFT_SHIFT,      		// D
+	KEY_E|MOD_LEFT_SHIFT,      		// E
+	KEY_F|MOD_LEFT_SHIFT,      		// F
+	KEY_G|MOD_LEFT_SHIFT,      		// G
+	KEY_H|MOD_LEFT_SHIFT,      		// H
+	KEY_I|MOD_LEFT_SHIFT,      		// I
+	KEY_J|MOD_LEFT_SHIFT,      		// J
+	KEY_K|MOD_LEFT_SHIFT,      		// K
+	KEY_L|MOD_LEFT_SHIFT,      		// L
+	KEY_M|MOD_LEFT_SHIFT,      		// M
+	KEY_N|MOD_LEFT_SHIFT,      		// N
+	KEY_O|MOD_LEFT_SHIFT,      		// O
+	KEY_P|MOD_LEFT_SHIFT,      		// P
+	KEY_Q|MOD_LEFT_SHIFT,      		// Q
+	KEY_R|MOD_LEFT_SHIFT,      		// R
+	KEY_S|MOD_LEFT_SHIFT,      		// S
+	KEY_T|MOD_LEFT_SHIFT,      		// T
+	KEY_U|MOD_LEFT_SHIFT,      		// U
+	KEY_V|MOD_LEFT_SHIFT,      		// V
+	KEY_W|MOD_LEFT_SHIFT,      		// W
+	KEY_X|MOD_LEFT_SHIFT,      		// X
+	KEY_Y|MOD_LEFT_SHIFT,      		// Y
+	KEY_Z|MOD_LEFT_SHIFT,      		// Z
+	KEY_IT_EGRAVE|MOD_RIGHT_ALT,	// [
+	KEY_IT_BACKSLASH,         		// bslash
+	KEY_IT_PLUS|MOD_RIGHT_ALT,		// ]
+	KEY_IT_IGRAVE|MOD_LEFT_SHIFT,   // ^
+	KEY_IT_MINUS|MOD_LEFT_SHIFT,   	// _
+	KEY_IT_QUOTE|MOD_RIGHT_ALT,    	// ` (I think this only works on Linux)
+	KEY_A,          		// a
+	KEY_B,          		// b
+	KEY_C,          		// c
+	KEY_D,          		// d
+	KEY_E,          		// e
+	KEY_F,          		// f
+	KEY_G,          		// g
+	KEY_H,          		// h
+	KEY_I,          		// i
+	KEY_J,          		// j
+	KEY_K,         		 	// k
+	KEY_L,          		// l
+	KEY_M,          		// m
+	KEY_N,          		// n
+	KEY_O,          		// o
+	KEY_P,          		// p
+	KEY_Q,          		// q
+	KEY_R,          		// r
+	KEY_S,          		// s
+	KEY_T,          		// t
+	KEY_U,          		// u
+	KEY_V,          		// v
+	KEY_W,          		// w
+	KEY_X,          		// x
+	KEY_Y,          		// y
+	KEY_Z,          		// z
+	KEY_IT_EGRAVE|MOD_LEFT_SHIFT|MOD_RIGHT_ALT,  // {
+	KEY_IT_BACKSLASH|MOD_LEFT_SHIFT,    // |
+	KEY_IT_PLUS|MOD_LEFT_SHIFT|MOD_RIGHT_ALT,  // }
+	KEY_IT_IGRAVE|MOD_RIGHT_ALT,	    // ~ (Linux only: This works in X11, in console it's KEY_0|MOD_RIGHT_ALT)

```

Comment:
Italiens do not have a tilde? O.o

Reply:
Unfortunately we don't, and we don't have a backtick either. At least Linux has some easy combinations for those (even though it's always annoyed me that console and X11 use different ones, grrrrr!), but on Windows you get into crappy ALT+numpad stuff, which is a nightmare on notebooks :/.

Reply:
Thatswhy I am using US layout for coding instead of german layout ;-)